### PR TITLE
Add TSC meeting notes for 11-30-2020

### PR DESCRIPTION
**Pull request by michdolan**
Signed-off-by: Michael Dolan <michdolan@gmail.com>

### CODE REVIEW

File: ASWF/meetings/tsc/2020-11-30.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  * [x] Alessandra Tomassi - Industrial Light & Magic
+
+# **OCIO TSC Meeting Notes**
+
+* Doug to recap goals of OCIO v2 and relation to OCIO mission:
+    - Some recent discussions about controversial topics. Thought it would be 
+      useful to recap what we're doing with v2 from AD point of view - clarify 
+      motivation for some of these new features. As written in origin proposal, 
+      goal is to be able to use OCIO across apps more broadly, and to rely on 
+      it entirely as out CMS. We would like to standardize on OCIO across apps.
```
Comment (doug-walker) [inline ASWF/meetings/tsc/2020-11-30.md:35]: out --> our

File: ASWF/meetings/tsc/2020-11-30.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+    - Some recent discussions about controversial topics. Thought it would be 
+      useful to recap what we're doing with v2 from AD point of view - clarify 
+      motivation for some of these new features. As written in origin proposal, 
+      goal is to be able to use OCIO across apps more broadly, and to rely on 
+      it entirely as out CMS. We would like to standardize on OCIO across apps. 
+      Need OCIO to do a variety of color management tasks. When thinking about 
+      how OCIO is used, there's color space based transformations 
+      (OCIOColorSpaceNode, etc.) and other operations which are more ignorant 
+      of color spaces (OCIOFileTransform, etc.). In ideal world, only the 
+      former would exist, since it requires the last knowledge from user.
```
Comment (doug-walker) [inline ASWF/meetings/tsc/2020-11-30.md:40]: last --> least

File: ASWF/meetings/tsc/2020-11-30.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+      can assign categories to UI elements and only show those. Don't think 
+      this should be OCIO-wide env var. Should be per-DCC.
+    - Doug: Yes, would be fixed for certain menu. Specific documented category 
+      so config authors know what an app uses.
+    - Mark B: Could end up with overlapping categories. If up to user they 
+      could define that themselves.
+    - Doug: Proposing standard categories to encourage consistency, but doesn't 
+      need to be limited to this. The key part is there is a set pre-defined to 
+      an application. So Maya would say what categories it's using for different 
+      menus. Would expect to find these at runtime.
```
Comment (doug-walker) [inline ASWF/meetings/tsc/2020-11-30.md:189]: Would expect to find these at runtime. --> There is another set that is end-user defined and would expect to find those at runtime.

**Review by doug-walker — APPROVED** [review]

File: ASWF/meetings/tsc/2020-11-30.md (hunk 1 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2020-11-30.md]: Fixed.

File: ASWF/meetings/tsc/2020-11-30.md (hunk 2 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2020-11-30.md]: Fixed.

File: ASWF/meetings/tsc/2020-11-30.md (hunk 3 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2020-11-30.md]: Fixed.

File: ASWF/meetings/tsc/2020-11-30.md [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+      Need OCIO to do a variety of color management tasks. When thinking about 
+      how OCIO is used, there's color space based transformations 
+      (OCIOColorSpaceNode, etc.) and other operations which are more ignorant 
+      of color spaces (OCIOFileTransform, etc.). In ideal world, only the 
+      former would exist, since it requires the least knowledge from user. 
+      Reality is though there is need for people to apply independent 
+      transforms, in apps where it's expected the user knows a reasonable 
+      amount about color management (e.g. Flame - users know how to load a LUT 
+      and apply it). Don't want two separate libs for applying LUTs. For things 
+      like dynamic properties and NamedTransform, it's more the later case,
```
Comment (Simran-B) [inline ASWF/meetings/tsc/2020-11-30.md:45]:
```suggestion
      like dynamic properties and NamedTransform, it's more the latter case, 
```

File: ASWF/meetings/tsc/2020-11-30.md [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+  * [x] Alessandra Tomassi - Industrial Light & Magic
+
+# **OCIO TSC Meeting Notes**
+
+* Doug to recap goals of OCIO v2 and relation to OCIO mission:
+    - Some recent discussions about controversial topics. Thought it would be 
+      useful to recap what we're doing with v2 from AD point of view - clarify 
+      motivation for some of these new features. As written in origin proposal, 
+      goal is to be able to use OCIO across apps more broadly, and to rely on 
+      it entirely as our CMS. We would like to standardize on OCIO across apps.
```
Comment (Simran-B) [inline ASWF/meetings/tsc/2020-11-30.md:35]:
```suggestion
      it entirely as our color management system (CMS).
      We would like to standardize on OCIO across apps. 
```

File: ASWF/meetings/tsc/2020-11-30.md (hunk 4 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2020-11-30.md]: Thanks. Fixed.

File: ASWF/meetings/tsc/2020-11-30.md (hunk 5 above)
Comment (michdolan) [reply ASWF/meetings/tsc/2020-11-30.md]: Good idea. Updated.

**Pull request merged**